src/util/domain-icon.js

```diff
@@ -38,6 +38,9 @@ export default function domainIcon(domain, state) {
     case 'input_select':
       return 'mdi:format-list-bulleted';
 
+    case 'input_slider':
+      return 'mdi:volume-high';
```

Comment:
mdi:ray-vertex seems like a good choice, didn't see that one when I was looking for a suitable icon.